api/Document.json

```diff
@@ -7546,17 +7628,11 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/onvisibilitychange",
           "support": {
-            "chrome": [
-              {
-                "version_added": "33"
-              },
-              {
-                "version_added": "13",
-                "prefix": "webkit"
-              }
-            ],
+            "chrome": {
+              "version_added": "62"
```

Comment:
This seems much too new. I suspect a similar situation as in https://github.com/mdn/browser-compat-data/pull/6849, can you break out this change?